### PR TITLE
utils/errors: remove cruft

### PR DESCRIPTION
**Pull request by mfojtik**

### CODE REVIEW

File: pkg/oc/lib/describe/projectstatus.go [hunk 1]
```diff
@@ -1984,7 +1984,10 @@ type bcLoader struct {
 func (l *bcLoader) Load() error {
 	list, err := l.lister.BuildConfigs(l.namespace).List(metav1.ListOptions{})
 	if err != nil {
-		return errors.TolerateNotFoundError(err)
+		if errors.IsNotFound(err) {
```
Comment (deads2k) [inline pkg/oc/lib/describe/projectstatus.go]: nit, I prefer this check above the first if

**Comment by mfojtik** [comment]
fixed conflict, applying lgtm label back.

File: pkg/oc/cli/idle/idle.go [hunk 2]
```diff
@@ -417,7 +417,7 @@ func findScalableResourcesForEndpoints(endpoints *corev1.Endpoints, getPod func(
 	controllerRefs := make(map[namespacedCrossGroupObjectReference]struct{})
 	for controllerRef := range immediateControllerRefs {
 		controller, err := getController(controllerRef)
-		if utilerrors.TolerateNotFoundError(err) != nil {
+		if err != nil && errors.IsNotFound(err) {
```
Comment (Miciah) [inline pkg/oc/cli/idle/idle.go:420]: @mfojtik, I think you lost a negation: 
```
 		if err != nil && !errors.IsNotFound(err) {
```
I noticed this while reviewing https://github.com/openshift/oc/pull/1026.

**Pull request merged**